src/js/routes/services.js

```diff
@@ -13,7 +13,7 @@ let serviceRoutes = {
   children: [
     {
       type: Route,
-      name: 'services-deployments',
+      name: '-services-deployments',
```

Comment:
`TabMixin` is expected to highlight tabs when their names appear within the route (for context, see https://github.com/dcos/dcos-ui/pull/153#issuecomment-218617058)

The upstream PR #153 tightens that requirement so that tabs are highlighted when their names appear at the start of the route. 

It's nice to reflect the route (`/services/deployments`) in the name, but doing that without some character prefixing the name causes the `TabMixin` to highlight both `services` and `services-deployments`. Adding a character prefix is the least hacky way I could come up with which allowed me to reflect the route in the name.